### ISSUE TITLE
download only if song does not exist

### ISSUE THREAD

**Issue by kmille** [kmille/deezer-downloader#15]
deezer: erst schauen obs schon da ist vor dem download (mal mit fdups nach duplicaten suchen)

**Comment by redmanmale** [comment]
As I can see [here](https://github.com/kmille/deezer-downloader/blob/68b08bb8e633981937bd7a597b88c8418d3a6ddc/app/music_backend.py#L93) this is already done.

**Comment by kmille** [comment]
Yep. But you can have more data sources than the Deezer Downloader downloads. The plan was to ask MPD if there is a song with the same name and same artist in the database.
